server/subscriptions.go

```diff
@@ -28,6 +29,27 @@ func (s *SubscriptionFlags) AddFlag(flag string) {
 	}
 }
 
+func (s SubscriptionFlags) String() string {
+	flags := []string{}
+	v := reflect.ValueOf(s)
+	typeOfS := reflect.TypeOf(s)
+
+	for i := 0; i < v.NumField(); i++ {
+		if !v.Field(i).Bool() {
```

Comment:
Do we _need_ to use `reflect` here? Could this be done without it?

Reply:
I thought about it, but this approach felt the cleanest, assuming we have to retain scope for more flags without changing the core `SubscriptionFlags` struct. Let me know if the assumption can be done away with or if you have a better approach.

Reply:
Using the [Go Proverbs](https://go-proverbs.github.io/) here I don't think using `reflect` is clear.

I had something like this in mind:

```golang
func (s SubscriptionFlags) String() string {
	flags := []string{}

	if s.ExcludeOrgMembers {
		flag := "--" + EXCLUDE_ORG_MEMBER_FLAG
		flags = append(flags, flag)
	}

	return strings.Join(flags, ",")
}
```